Classes/GPXType.swift

```diff
@@ -188,7 +188,7 @@ open class GPXType: NSObject {
     
     func value(forDateTime date: Date?) -> String? {
         let formatter = DateFormatter()
-        formatter.timeZone = TimeZone(secondsFromGMT: 0)
+        formatter.timeZone = GPXParser.timezone
```

Comment:
I am slowly trying to depreciate the entire class `GPXType`, or depreciate parts of it, which as you can see from most of the code, the class here is hardly updated.